be1-go/popcha/server_test.go

```diff
@@ -144,13 +163,43 @@ func TestAuthRequestFails(t *testing.T) {
 	log.Info().Msg(partialURL)
 	require.NoError(t, err)
 
+	lastEntry = logTester.LastEntry()
+	lastEntry.ExpLevel(zerolog.ErrorLevel)
+	lastEntry.ExpMsg("Error while verifying the parameters of the request")
 	bodyBytes = helperValidateGetAndParseBody(t, res)
-
 	// we require that there are 5 missing arguments
 	helperMissingArgs(t, bodyBytes, 5)
 
+	time.Sleep(200 * time.Millisecond)
```

Comment:
isn't there another way to wait ? sleep produces long tests. Plus all the non deterministic problems.

Reply:
I can definitely remove most of those.